### PR TITLE
Feat/revise forms and cells

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

**Review by robertu7 — CHANGES_REQUESTED** [review]

**Comment by robertu7** [comment]
Cell is a bit weird on login/signup form...can you confirm with designer?

image

File: src/components/Form/List/styles.css [hunk 1]
```diff
@@ -11,33 +11,48 @@
 }
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/Form/List/styles.css:11]: Next PR

File: src/common/styles/mixins.css [hunk 2]
```diff
@@ -119,47 +119,39 @@
...
   padding: var(--spacing-base);
   font-size: var(--font-size-md);
+  color: var(--color-grey-dark);
   background-color: var(--color-white);
+  border-radius: var(--spacing-x-tight);
+
+  &:focus,
+  &.focus {
+    color: var(--color-black);
+    border-color: var(--color-grey);
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/common/styles/mixins.css:134]: done

File: src/components/Form/List/styles.css [hunk 3]
```diff
@@ -47,7 +62,7 @@ ul {
     }
 
     & :global(> li + li) {
-      border-top: 1px solid var(--color-white);
+      border-top: 1px dashed var(--color-grey-light);
     }
   }
 }
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/Form/List/styles.css:68]: Next PR

File: src/views/Me/Wallet/styles.css [hunk 4]
```diff
@@ -6,18 +6,17 @@
     margin-bottom: var(--spacing-loose);
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/views/Me/Wallet/styles.css:6]: NEXT PR

File: src/common/styles/mixins.css [hunk 5]
```diff
@@ -119,47 +119,39 @@
 /* Form
    ========================================================================== */
 @define-mixin form-input {
-  @mixin border-top-grey;
-  @mixin border-bottom-grey;
+  @mixin border-grey;
 
   padding: var(--spacing-base);
   font-size: var(--font-size-md);
+  color: var(--color-grey-dark);
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/common/styles/mixins.css]: yes, still black and the grey-dark is for placeholder,

File: src/components/Form/List/Item/index.tsx [hunk 6]
```diff
@@ -50,8 +50,9 @@ const Item: React.FC<ItemProps> = forwardRef(
       top: leftAlign === 'top',
       bold: !!bold,
     })
+    const [hoverRef, isHovered] = useHover<HTMLDivElement>()
     const itemContent = (
-      <section className="container">
+      <section className="container" ref={hoverRef}>
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/Form/List/Item/index.tsx]: done

File: src/components/Form/List/styles.css [hunk 7]
```diff
@@ -11,33 +11,48 @@
...
   & :global(> li + li) {
-    border-top: none;
+    border-top: 1px dashed var(--color-grey-light);
   }
 
-  &.grey {
-    margin-right: var(--spacing-base);
-    margin-left: var(--spacing-base);
-    border-radius: var(--spacing-x-tight);
+  /* Fix <DropdownDialog /> contain <Form.List.Item />, when click dropdown
```
[the remark this answers is not in the document]
Comment (robertu7) [reply src/components/Form/List/styles.css:28]: @pitb2022 please take a look at this comment

**Review by robertu7 — APPROVED** [review]

**Pull request merged**